packages/web-react/src/hooks/useStyleUtilities.ts

```diff
@@ -0,0 +1,51 @@
+import { SpacingStyleProp } from '../constants';
+import { BreakpointToken, SpaceToken, StyleProps } from '../types';
+import { applyClassNamePrefix } from '../utils';
+
+export type StyleUtilitiesResult = {
+  styleUtilities: string[];
+  props: StyleProps;
+};
+
+const normalizeSpacingValue = (value: string): string => (value === 'auto' ? 'auto' : value.replace(/[^0-9]/g, ''));
+
+const processBreakpointProperties = (
+  utilityName: string,
+  propValue: Partial<Record<BreakpointToken, SpaceToken | 'auto'>>,
+  prefix: string | null | undefined,
+  accumulatedUtilities: string[] = [],
```

Comment:
> accumulatedUtilities

This sounds like `reduce()` should be used instead of `forEach()`. `accumulatedUtilities` shouldn't be necessary at all.